packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -415,6 +410,7 @@ class CupertinoDatePicker extends StatefulWidget {
   final Color? backgroundColor;
 
   /// Whether to show day of week alongside day. Defaults to false.
+  /// Only works with [CupertinoDatePickerMode.date].
```

Comment:
An opportunity to improve the documentation here.

```suggestion
  /// Whether to show the day of week alongside the day in [CupertinoDatePickerMode.date] mode.
  ///
  /// Defaults to false.
```